docs/source/reference/rest_api.rst

```diff
@@ -0,0 +1,1265 @@
++-----------------------------------------------------+-------------------------------------+------------------+
+| ``name=ilike="a%d_"``                               |   "aiida", "AiiDA"                  |                  |
++-----------------------------------------------------+-------------------------------------+------------------+
+| ``name=like="a_d_"``                                |                                     |     "aiida"      |
++-----------------------------------------------------+-------------------------------------+------------------+
+| ``name=like="aii%d_a"``                             |        "aiida"                      |                  |
++-----------------------------------------------------+-------------------------------------+------------------+
+| ``uuid=like="cdfd48%"``                             |"cdfd48f9-7ed2-4969-ba06-09c752b83d2"|                  |
++-----------------------------------------------------+-------------------------------------+------------------+
+|``description=like="This calculation is %\% useful"``|"This calculation is 100% useful"    |                  |
++-----------------------------------------------------+-------------------------------------+------------------+
+
+The membership operator ``=in=`` has to be followed by a comma-separated list of values of the same type.
+The condition is fulfilled if the column value of an object is an element of the list.
+
+Examples::
+
+    http://localhost:5000/api/v4/nodes?id=in=45,56,78
+    http://localhost:5000/api/v4/computers/?scheduler_type=in="slurm","pbs"
+
+The relational operators '<', '>', '<=', '>=' assume natural ordering for integers, (case-insensitive) alphabetical
+ordering for strings, and chronological ordering for datetime values.
+
+Examples:
+
+    - ``http://localhost:5000/api/v4/nodes?id>578`` selects the nodes having an id larger than 578.
+    - ``http://localhost:5000/api/v4/users/?last_name<="m"`` selects only the users whose last name begins with a
+      character in the range [a-m].
+
+
+.. note:: Node types have to be specified by a string that defines their position in the AiiDA source tree ending
+    with a dot. Examples:
+
+    - ``node_type="data.code.Code."`` selects only objects of *Code* type
+    - ``node_type="data.remote.RemoteData."`` selects only objects of *RemoteData* type
+
+.. note:: If you use in your request the endpoint *links/incoming* (*links/outgoing*) together with one or more filters, the
+    latter are applied to the incoming (outgoing) nodes of the selected *id*. For example, the request:
+
+        ::
+
+            http://localhost:5000/api/v4/nodes/a67fba41/links/outgoing?full_type="data.dict.Dict.|"
+
+    would first search for the outgoing of the node with *uuid* starting with "a67fba41" and then select only those
+    nodes of full_type *data.dict.Dict.|*.
+
+
+
+Deployment on a web server
+==========================
```

Comment:
Is this whole section not better placed elsewhere? It is not strictly reference. Is this intended for users? As in, this is a way that we recommend users serve the REST API?

Reply:
> Is this whole section not better placed elsewhere? It is not strictly reference.

In the TOC google doc, I mentioned that I think this could be a "howto" as well but I don't see it fit into any of the existing categories.

> Is this intended for users? 

Yes, for a certain subset of users who might want to serve their REST API on a web server (not just for personal use)

Reply:
What about adding a section to the "How to work with data" at the end "Serving data over the REST API" or something to that effect. We will have explained the use of the REST API for personal use in the "Finding data" section before that. At the end we can also add a reference

Reply:
I've followed your advice and added a section to "managing data" ("Serving your data to others")